extensions/laravel-livewire/src/documentation/2.x.json

```
@@ -0,0 +1,49 @@
+{
+  "Start": [
+    { "url": "https://laravel-livewire.com/docs/2.x/quickstart", "title": "Quickstart2" },
```

```suggestion
    { "url": "https://laravel-livewire.com/docs/2.x/quickstart", "title": "Quickstart" },
```
Remove debug number 😄